bin/ConcordanceCheck.sh

```diff
@@ -233,16 +233,18 @@ set -eu
 	bgzip -c "${concordanceDir}/tmp/${concordanceCheckId}/${arrayId}.FINAL.ExonFiltered.vcf" > "${concordanceDir}/tmp/${concordanceCheckId}/${arrayId}.FINAL.ExonFiltered.vcf.gz"
 	tabix -p vcf "${concordanceDir}/tmp/${concordanceCheckId}/${arrayId}.FINAL.ExonFiltered.vcf.gz"
 
-	java -XX:ParallelGCThreads=1 -Djava.io.tmpdir="${concordanceDir}/temp/" -Xmx9g -jar ${EBROOTCOMPAREGENOTYPECALLS}/CompareGenotypeCalls.jar \
-	-d1 "${concordanceDir}/tmp/${concordanceCheckId}/${arrayId}.FINAL.ExonFiltered.vcf.gz" \
-	-D1 VCF \
-	-d2 "${concordanceDir}/tmp/${concordanceCheckId}/${ngsId}.FINAL.vcf.gz" \
-	-D2 VCF \
-	-ac \
-	--sampleMap "${sampleSheet}" \
-	-o "${concordanceDir}/tmp/${concordanceCheckId}" \
+	java -XX:ParallelGCThreads=1 -Djava.io.tmpdir="${concordanceDir}/temp/" -Xmx9g -jar ${EBROOTCOMPAREGENOTYPECALLS}/CompareGenotypeCalls.jar \\
+	-d1 "${concordanceDir}/tmp/${concordanceCheckId}/${arrayId}.FINAL.ExonFiltered.vcf.gz" \\
```

Comment:
Just curious: why do we need double slashes (```\\```) at the end of the lines here?

Reply:
omdat je dan in de gecreerde jobs ook regel breaks hebt, anders is het 1 regel.

Reply:
Aha, weer wat geleerd :)